examples/fscloud/README.md

```diff
@@ -5,7 +5,7 @@
 An example using the fscloud profile to deploy a compliant Postgresql instance. This example uses the IBM Cloud terraform provider to:
 
 - Create a new resource group if one is not passed in.
-- Create a new ICD Postgresql database instance and credentials.
+- Create a new ICD Postgresql database instance with and credentials.
```

Comment:
No need for `and`

Reply:
I am removing `with` to make it same as earlier

I had added `with` different intention but later changed mind